server/__init__.py

```diff
@@ -275,9 +275,9 @@ def create_app():
   if app.config['LOCAL']:
     os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
 
-  if app.config['API_PROJECT']:
+  if cfg.SECRET_PROJECT:
```

Comment:
Will all secrets (currently there's maps api key and oauth client id) always be stored in the same project? oauth client and maps secrets will always be local to the custom dc, so meaning mixer api keys will always be in custom dc project as well? Just making sure

Reply:
Yes, for custom DC, both api keys and the secret manager is within the custom project.